easybuild/easyconfigs/o/OpenForceField-Toolkit/OpenForceField-Toolkit-0.16.0-foss-2023a.eb

```diff
@@ -0,0 +1,144 @@
+    }),
+    ('python-constraint', '1.4.0', {
+        'modulename': 'constraint',
+        'source_tmpl': '%(name)s-%(version)s.tar.bz2',
+        'checksums': ['501d6f17afe0032dfc6ea6c0f8acc12e44f992733f00e8538961031ef27ccb8e'],
+    }),
+    ('bson', '0.5.10', {
+        'checksums': ['d6511b2ab051139a9123c184de1a04227262173ad593429d21e443d6462d6590'],
+    }),
+    ('xmltodict', '0.13.0', {
+        'checksums': ['341595a488e3e01a85a9d8911d8912fd922ede5fecc4dce437eb4b6c8d037e56'],
+    }),
+    ('intermol', '0.1.2', {
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/shirtsgroup/InterMol/archive/'],
+        'checksums': ['a3065133ba16585a5db3807466a36242d191156525c33079cf46a97cedcfe835'],
+    }),
+    ('SMIRNOFF99Frosst', '1.1.0', {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/openforcefield/smirnoff99Frosst/archive/'],
+        'checksums': ['940c4ed3cd720e746549a3335f10b7b0d9047a67bc216ad45dd875be5c8ec467'],
+    }),
+    ('SMIRNOFF-Plugins', '2024.01.0', {
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/openforcefield/smirnoff-plugins/archive/'],
+        'checksums': ['8ba4dbc6ed320f06f30bf42e8f70399bc3818becd668990693b51ddca183ff74'],
+    }),
+    ('OpenForceField-ForceFields', '2024.04.0', {
+        'modulename': 'openforcefields',
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/openforcefield/openff-forcefields/archive/'],
+        'checksums': ['fc7094cdb74bf9f0ed665dfc716cc62c5a37def3fc7e852adeae0ae4b60169c7'],
+    }),
+    ('OpenForceField-Amber-FF-Ports', '0.0.4', {
+        'modulename': 'openff.amber_ff_ports',
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/openforcefield/openff-amber-ff-ports/archive/'],
+        'checksums': ['dbf2d1ccce06b0627b1eccae7e227ad800ff1cbac3e869344de265b178cc0207'],
+    }),
+    ('OpenForceField-Utilities', '0.1.12', {
+        'modulename': 'openff.utilities',
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/openforcefield/openff-utilities/archive/'],
+        'checksums': ['92882c5b87b40bdb4ee1e8c84c0ecc0fc83f26eaf3e3befde41bfdcf93d75dea'],
+    }),
+    ('OpenForceField-Units', '0.2.2', {
+        'modulename': 'openff.units',
+        'source_tmpl': '%(version)s.tar.gz',
+        'source_urls': ['https://github.com/openforcefield/openff-units/archive/'],
+        'checksums': ['532746534c1937938580d5cfe98f414aff9bfbcc66970e115a88a38f6d5a2581'],
+    }),
+    ('OpenForceField-NAGL-Models', '0.2.0', {
+        'modulename': 'openff.nagl_models',
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/openforcefield/openff-nagl-models/archive/'],
+        'checksums': ['df8e7ee980aa7ea99559759258a16e3aaa362245f5b7c32f9c25fb226eaee222'],
```

Comment:
I'm getting a different checksum here now when testing: `5ccc55d7f0fcd24577796fd8b2bfd85b035ee8060e441c1f7dfc6f1949b9dee5`.

Checking the differences between the old source tarball, which was downloaded on `generoso` on 19 July'24, and the one we get now, I see:

```diff
diff -ru openff-nagl-models-0.2.0.20240719/openff/nagl_models/_version.py openff-nagl-models-0.2.0.20240806/openff/nagl_models/_version.py
--- openff-nagl-models-0.2.0.20240719/openff/nagl_models/_version.py	2024-04-19 11:13:35
+++ openff-nagl-models-0.2.0.20240806/openff/nagl_models/_version.py	2024-04-19 11:13:35
@@ -26,7 +26,7 @@
     # setup.py/versioneer.py will grep for the variable names, so they must
     # each be defined on a line of their own. _version.py will just call
     # get_keywords().
-    git_refnames = " (HEAD -> main, tag: v0.2.0)"
+    git_refnames = " (tag: v0.2.0)"
     git_full = "25a63cd83a667578447f6c4c745ab07a6209bef2"
     git_date = "2024-04-19 19:13:35 +1000"
     keywords = {"refnames": git_refnames, "full": git_full, "date": git_date}
```

Next to this required fix, I would also like to see us consistently using `sources` rather than `source_tmpl`, and renaming the downloaded source tarballs so you can easily tell what's what, like for example:

```python
    ('OpenForceField-NAGL-Models', '0.2.0', {
        'modulename': 'openff.nagl_models',
        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
        'source_urls': ['https://github.com/openforcefield/openff-nagl-models/archive/'],
        'checksums': ['5ccc55d7f0fcd24577796fd8b2bfd85b035ee8060e441c1f7dfc6f1949b9dee5'],
    }),
```